controllers/conditions.go

```diff
@@ -71,6 +71,12 @@ func (r *ImageClusterInstallReconciler) initializeConditions(ctx context.Context
 			Reason:  "Unknown",
 			Message: "Unknown",
 		},
+		{
+			Type:    hivev1.ClusterInstallConditionType(v1alpha1.HostValidatedCondition),
```

Comment:
What's the difference between `Validated` and `RequirementsMet`?

If we can make this logic fit in one of the hive API conditions that would be better since it would then be copied to the cluster deployment.

Reply:
We use RequirementsMet to set Done for image, in this case it will be confusing to set RequirementsMet success and then after adding bmh reference to set it to failed. That's why i added new condition

Reply:
Do we really need a separate condition for each? I think it makes sense to set requirements met to success when we don't have a BMH, but this validation is also a good match for pre-install cluster requirements.

Or, put another way, we already would use a "validated" condition to mean a bunch of possible issues why not apply that same logic to the condition we already have?

Reply:
Also site config is only going to copy the hive contract conditions back to the site config status.

So those users will have to look at the ImageClusterInstall specifically to see this condition.

Reply:
To say the truth i have no objections, though it means that there will be some difference for same condition with and without bmh, but sounds like it is not a big deal

Reply:
The condition message can reflect the difference.